### PR TITLE
[editor] Center to map button.

### PR DESCRIPTION
**Pull request by summetdev**
This pull request implements [#387](https://github.com/Anuken/Mindustry-Suggestions/issues/387) suggestion.

**Preview:**
preview-gif

* [x] English & Russian translations.
* [x] Code tested.

### CODE REVIEW

**Comment by DeltaNedas** [comment]
good

File: core/src/mindustry/editor/MapView.java [hunk 1]
```diff
@@ -173,6 +173,10 @@ public void setGrid(boolean grid){
         this.grid = grid;
     }
 
+    public void center() {
```
Comment (Quezler) [inline core/src/mindustry/editor/MapView.java:176]:
```suggestion
    public void center(){
```

**Comment by sk7725** [comment]
We also need something that shows the direct center of the map

**Comment by sk7725** [comment]
Maybe a crosshair?

**Comment by DeltaNedas** [comment]
@Anuken update?

**Comment by unjown** [comment]
> We also need something that shows the direct center of the map

@Anuken this one seems better to me

**Comment by Anuken** [comment]
Implemented slightly differently.

> Maybe a crosshair?

You can see the map center by enabling the grid.

**Comment by unjown** [comment]
> Implemented slightly differently.
> 
> > Maybe a crosshair?
> 
> You can see the map center by enabling the grid.

?  grid? Make it glow pls the mid i mean the grid

**Pull request closed without merging**